manager/integration/tests/test_node.py

```diff
@@ -2239,4 +2256,282 @@ def test_node_eviction():
     the same as checksum 1 and checksum 2.
     28. Set 'Eviction Requested' to 'false' and enable scheduling on node 2.
     """
-    pass
+    nodes = client.list_node()
+
+    node1 = nodes[0]
+    node2 = nodes[1]
+    node3 = nodes[2]
+    client.update(node3, allowScheduling=False)
+
+    volume1_name = volume_name + "-1"
+    volume1_size = str(500 * Mi)
+    volume1_data_path = "/data/test"
+    pv1_name = volume1_name + "-pv"
+    pvc1_name = volume1_name + "-pvc"
+    pod1_name = volume1_name + "-pod"
+    pod1 = copy.deepcopy(pod)
+
+    pod1['metadata']['name'] = pod1_name
+    pod1['spec']['volumes'] = [{
+        "name": "pod-data",
+        "persistentVolumeClaim": {
+            "claimName": pvc1_name
+        }
+    }]
+
+    pod1['spec']['nodeSelector'] = \
+        {"kubernetes.io/hostname": node3.name}
```

Comment:
I think this will mount the volume-1 on node 3, right? Can we change it to node1 to align with the test skeleton. Thanks.

Reply:
fixed